src/borg/archiver.py

```diff
@@ -3211,9 +3211,14 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
         are meaning different things:
 
         This archive / deduplicated size = amount of data stored ONLY for this archive
-                                         = unique chunks of this archive.
+        = unique chunks of this archive.
         All archives / deduplicated size = amount of data stored in the repo
-                                         = all chunks in the repository.
+        = all chunks in the repository.
+
+        Borg only supports archives up to a certain size. The size of an archive
```

Comment:
well, this is correct, but can be misunderstood by users who do not know that content data lives outside of the archive. so maybe rather say "... archive metadata streams up to ...".